### PR TITLE
Compatibility with Rails 5.0

### PR DESCRIPTION
**Pull request by pschambacher**
:octopus::computer:

/cc @grosser @bquorning @hsume2 @osheroff 

### Steps to merge
* [ ] :+1: from the team

### Risks
* None

### CODE REVIEW

File: gemfiles/rails3.2.gemfile [hunk 1]
```diff
@@ -1,8 +1,5 @@
-source "https://rubygems.org"
-
-gemspec :path=>"../"
-
-gem "activerecord", "~> 3.2.18"
-gem "minitest", "~> 4.7.0"
-
 eval(File.read('gemfiles/common.rb'))
```
Comment (bquorning) [inline gemfiles/rails3.2.gemfile]: `eval_gemfile` while you’re at it?

Comment (pschambacher) [inline gemfiles/rails3.2.gemfile]: Ah true

**Comment by pschambacher** [comment]
:haircut:

**Comment by grosser** [comment]
adding into `Assocition::VALID_OPTIONS` sounds good to me too ... why did you not like that ?

**Comment by grosser** [comment]
:+1: anyway ...

**Comment by pschambacher** [comment]
Modifying a constant... might not be a great idea

**Comment by grosser** [comment]
meh ... overwriting a method that reads that constant might not be a good idea either ;)

**Comment by grosser** [comment]
does not matter ... do what your heart tells you and then merge :)

**Comment by pschambacher** [comment]
I'll just leave a chance for the others to express their opinion. I'll advise when I come in tomorrow morning :smile:

**Pull request merged**